products.js

```diff
@@ -142,21 +143,21 @@ function tunedImageFnodes() {
     // return tuningRoutine;
 }
 
-function tunedTitleFnodes() {
-    // const rules = ruleset(
-    //   //get all title tags in the inserted fixture
-    //   rule(dom('div#fixture > title'), type('titleish')),
-    //
-    //   //return image with max score
-    //   rule(type('titleish').max(), out('product-title'))
-    //
-    // );
-    //
-    // function tuningRoutine(doc) {
-    //     return rules.against(doc).get('product-title');
-    // }
-    //
-    // return tuningRoutine;
+function tunedTitleFnodes(dict) {
+    const rules = ruleset(
+      //get all title tags in the inserted fixture
+      rule(dom('title'), type('titleish')),
+
+      //return image with max score
+      rule(type('titleish').max(), out('product-title'))
```

Comment:
Yeah, I've found that there's usually only one title tag per page, so fathom probably wasn't necessary for this? I just did it anyways